node/tests/financials_test.rs

```diff
@@ -84,16 +98,23 @@ fn financials_command_retrieves_payable_and_receivable_records() {
     assert_eq!(payable[0].wallet, wallet_payable.to_string());
     assert_eq!(payable[0].balance_gwei, 45678357_u64);
     assert_eq!(payable[0].pending_payable_hash_opt, None);
-    assert_eq!(receivable[0].wallet, wallet_receivable_1.to_string());
-    assert_eq!(receivable[0].balance_gwei, 9000_i64);
-    assert_eq!(receivable[1].wallet, wallet_receivable_2.to_string());
-    assert_eq!(receivable[1].balance_gwei, 345678_i64);
```

Comment:
We can't assert this way, because we can't depend on the order of the receivables. So we first have to figure out which order they're in, and _then_ assert on them.